hack/jenkins/e2e.sh

```diff
@@ -164,6 +166,7 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
+    "GCE\sL7\sLoadBalancer\sController"
```

Comment:
I just put in a quota request for more forwarding rules, though to make this parallel I'll have to bump it up even more, since we're currently at 100% forwarding rule usage (maybe an existing leak). The namespace doesn't matter in this case, as the L7 controller is running as a cluster addon (so it creates lbs for all namspaces) and the quota limitations are global. 

I'll add a todo